### ISSUE TITLE
alertdialog role - autofocus wrong???; missing check for aria-label | aria-labelledby

### ISSUE THREAD

**Issue by icecream17** [jsx-eslint/eslint-plugin-jsx-a11y#816]
For alertdialog, it says

> When the alert dialog is displayed, authors _____SHOULD_____ set focus to an active element within the alert dialog, such as a form control or confirmation button.
> https://raw.githack.com/w3c/aria/stable/#alertdialog

If `active element` means `any element` then autoFocus is allowed. But I don't really know what that sentence is saying, since ... they're saying to set focus to a focused element.

Also, (`aria-label` or `aria-labelled-by`) seems to be required:
> https://www.w3.org/TR/wai-aria-practices-1.1/#naming_role_guidance
